index.html

```diff
@@ -47,16 +54,16 @@
 			<link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet" />
 		</noscript>
 	</head>
-	<body class="nes-container">
+	<body>
 		<div class="container">
 			<br/>
 			<h1 class="nes-text"><strong> Rahul Zhade </strong></h1><br>
-			<h3 class="nes-text"> AppSec Engineer at <i class="nes-octocat animate"></i> </h3>
+			<h3 class="nes-text"> AppSec Engineer at <a class="nes-btn is-success" href="https://github.com"><i class="nes-octocat animate"></i></a> </h3>

```

Comment:
Dope. :octocat: 

nit: I played around with the CSS a bit and you can make the octocat box/button more "box-y" if you apply the following:

```css
a.nes-btn.is-success {
	display: inline-flex;
}

.nes-octocat {
	width: 92;
}
```
Result:
<p align="center">
	<img width=350 src="https://user-images.githubusercontent.com/8808097/100488387-c1508380-30c2-11eb-8f60-e626eb11c118.png" />
</p>